### PR TITLE
Add indexes to the migration files

### PR DESCRIPTION
**Pull request by DerekStride**
I isolated the indexes from #17 to make it easier to review.

Once this is merges you'll need to run the following command.

```bash
rails db:drop && rails db:create && rails db:migrate && rails db:seed
```

### CODE REVIEW

**Comment by DerekStride** [comment]
@ismail-syed @justinkrol 
Can I get some 👀 on this? It adds indexes and fixes the association names (from #21).

I also added the basic tests for the user model. I'll add more when #24 is merged.

**Review by ismail-syed — APPROVED** [review]
🚢

File: db/migrate/20170923230214_create_likes.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 class CreateLikes < ActiveRecord::Migration[5.1]
   def change
-    create_table :likes do |t|
-      t.integer :user_id, null: false
-      t.integer :content_id, null: false
-      t.string :content_type, null: false
+    create_table :likes, id: false do |t|
```
Comment (justinkrol) [inline db/migrate/20170923230214_create_likes.rb:3]: Why id false? If this is to enforce one like per user/content, why not just do uniqueness validations instead?

Comment (DerekStride) [reply db/migrate/20170923230214_create_likes.rb:3]: Why do we need an id? It's essentially just a join table.

**Pull request merged**